internal/terraform/context_plan2_test.go

```diff
@@ -6780,9 +6780,8 @@ resource "test_simple_object" "b" {
 }
 `,
 			},
-			expectedAttributes: []string{
-				"test_computed_object.a.object",
-				"test_computed_object.a.object.value",
+			expectedDiags: []string{
+				"Invalid index:The given key does not identify an element in this collection value. An object only supports looking up attributes by name, not by numeric index.",
```

Comment:
Nitpick: I assume long run we'd probably want a space there `Invalid index: The ...` but I don't think this is even worth debating now and blocking the release on it 😅